app/Services/TokenManager.php

```
@@ -0,0 +1,172 @@
+
+        $response = Http::post("{$this->baseUrl}/token/new/", [
+            'secret_id' => $this->user->gocardless_secret_id,
+            'secret_key' => $this->user->gocardless_secret_key,
+        ]);
+
+        if (!$response->successful()) {
+            throw new \Exception('Failed to get new token set: ' . $response->body());
+        }
+
+        $data = $response->json();
+        $this->updateTokens($data);
```

PHPStan (2.1.15)

Parameter #1 $tokenData of method App\Services\TokenManager::updateTokens() expects array, mixed given.


(argument.type)